localfs/store_test.go

```diff
@@ -111,3 +111,99 @@ func TestGetDir(t *testing.T) {
 	err = store.Delete(context.Background(), "test/index.html")
 	require.NoError(t, err)
 }
+
```

Comment:
BTW, if you add the test to this file: https://github.com/lytics/cloudstorage/blob/master/testutils/testutils.go

Then it will be run for all Store types, or a subset of them if you set up exclusions.  We add most of the tests there so that we can make sure the various backends have parity.   E.g. if we want them all to treat EndOffset as inclusive or exclusive...

Reply:
I saw that, but avoided it because the behavior is not universal to all stores. I didn't spot the mechanism you refer to for doing exclusions.

I'm also uneasy about the degree of sequential coupling within those tests, and wasn't eager to add to them unless I was going to devote more focus to avoiding new debt while doing so.